public/demo.html

```diff
@@ -11,30 +11,30 @@
     <h1>Humm widgets cards online</h1>
 
     <h2>AU default widget</h2>
-    <script src="./humm-widgets-cards-au.umd.js?productPrice=400"></script>
+    <script src="./humm-widgets-cards-au.umd.js?productPrice=400&merchantId='aaa143af-a96b-4887-bcd2-85960da544ea'"></script>
```

Comment:
Are these `merchantId`'s test values? Are they safe to leave visible in a public repo?

Reply:
these are randomly generated -> https://www.uuidgenerator.net/

Reply:
:phew: 👌